app/controllers/admin/chat_channels_controller.rb

```diff
@@ -1,5 +1,9 @@
 module Admin
   class ChatChannelsController < Admin::ApplicationController
```

Comment:
This is `ChatChannelsController`, which is not related to comments?

Reply:
Oh no .... 🤦‍♀️